HomeTasks/HomeTask3.cs

```diff
@@ -0,0 +1,19 @@
+namespace HomeTasks;
+//Write a program to determine whether the number specified by the user from 0 to 100 falls within the numerical range 
+//[0 - 14] [15 - 35] [36 - 50] [51 - 100]. If yes, indicate in which specific range. 
+//If the user specifies a number that does not belong to any of the existing numerical ranges, display the corresponding message.
+public class HomeTask3 : IBaseHomeTask
+{
+    public void Run()
```

Comment:
general note: use reformat document or hotkeys to validate files

Option + Command + L or Command + K + D